### PR TITLE
fix: moving campaign from selling to CRM

### PR DESCRIPTION
**Pull request by anupamvs**
Changing Campaign module Selling -> CRM

### CODE REVIEW

**Comment by ruchamahabal** [comment]
For the reviewer:

1. Check related imports
2. Check migration and installation on a new site
3. Check if a patch is required
4. Check if Desk Page updated

File: erpnext/crm/doctype/campaign/campaign.py [hunk 1]
```diff
@@ -1,11 +1,8 @@
-# Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
-# License: GNU General Public License v3. See license.txt
+# Copyright (c) 2021, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
 
-from __future__ import unicode_literals
 import frappe
-
 from frappe.model.document import Document
-from frappe.model.naming import set_name_by_naming_series
```
Comment (pateljannat) [inline erpnext/crm/doctype/campaign/campaign.py:8]: Import for set_name_by_naming_series has been removed but its still being called from autoname function

File: erpnext/crm/doctype/campaign/campaign.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2021, Frappe Technologies Pvt. Ltd. and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('Campaign', {
+	refresh: function(frm) {
+		erpnext.toggle_naming_series();
+
+		if(frm.doc.__islocal) {
+			frm.toggle_display("naming_series", frappe.boot.sysdefaults.campaign_naming_by=="Naming Series");
```
Comment (pateljannat) [inline erpnext/crm/doctype/campaign/campaign.js:9]: The field campaign_naming_by is still in the Selling settings. Should we create a CRM setting and move it there?

**Review by pateljannat — CHANGES_REQUESTED** [review]

File: erpnext/crm/doctype/campaign/campaign.js (hunk 2 above)
Comment (anupamvs) [reply erpnext/crm/doctype/campaign/campaign.js:9]: Yeah! CRM setting is not introduced yet. In my roadmap maybe in next month.

**Review by pateljannat — APPROVED** [review]

**Pull request merged**